tests/integration/test_enrollment/data/wazuh_enrollment_tests.yaml

```diff
@@ -0,0 +1,273 @@
+-
+  name: "Default Name"
+  description: "No agent name in auto enrollment configuration"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}'\n"
+    response: "OSSEC K:'001 {host_name} any TopSecret'\n"
+-
+  name: "Valid agent address"
+  description: "Check a valid agent_address configurations"
+  configuration:
+    agent_address: "192.168.0.10"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}' IP:'192.168.0.10'\n"
+    response: "OSSEC K:'001 {host_name} 192.168.0.10 TopSecret'\n"
+-
+  name: "Valid agent address CIDR"
+  description: "Check a valid agent_address configurations CIDR"
+  configuration:
+    agent_address: "192.168.0.10/24"
+  pre_existent_keys:
+    - ""
+  message:
+    expected: "OSSEC A:'{host_name}' IP:'192.168.0.10/24'\n"
+    response: "OSSEC K:'001 {host_name} 192.168.0.10/24 TopSecret'\n"
+-
+  name: "Invalid Agent address CIDR"
+  description: "Check a invalid CIDR agent_address in auto enrollment configuration"
+  configuration:
+    agent_address: "192.168.0.10/100"
+  expected_error: "ERROR: (4104): Invalid hostname: '192.168.0.10/100'."
+  #Known Issue: Invalid CIDR is not checked. Skiped
+  skips:
+    - agent-auth
+    - wazuh-agentd
+-
+  name: "Invalid Agent address"
+  description: "Check a invalid agent_address in auto enrollment configuration"
+  configuration:
+    agent_address: "192.168.0.1000"
+  pre_existent_keys:
+    - ""
+  expected_error: "ERROR: (4104): Invalid hostname: '192.168.0.1000'."
+  #Known Issue: Invalid adress is not checked. Skiped
+  skips:
+    - agent-auth
+    - wazuh-agentd
+-
+  name: "Valid manager address"
+  description: "Check a valid manager_address configurations"
```

Comment:
Configuration instead of configurations?

Reply:
Fixed